docs/index.md

```diff
@@ -24,19 +24,19 @@ Installing Postgres.app
 
 <ul class="instructions">
 	<li>
-		<p>Download &nbsp; ➜ &nbsp; Move to Applications folder &nbsp; ➜ &nbsp; Double Click</p>
+		<p>Download &nbsp; ➜ &nbsp; Move to Applications folder &nbsp; ➜ &nbsp; Double click to open</p>
 		<p class="subdued">If you don't move Postgres.app to the Applications folder, you will see a warning about an unidentified developer and won't be able to open it.</p>
```

Comment:
<p class="subdued">If you don't move Postgres.app to the Applications folder, you will see a warning about an unidentified developer and **you** won't be able to open it.</p>